substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -735,6 +735,17 @@ pub mod pallet {
 	#[pallet::storage]
 	pub(crate) type ChillThreshold<T: Config> = StorageValue<_, Percent, OptionQuery>;
 
+	/// The total amount of stake backed by the lowest third of validators for the last
+	/// upper bound eras. This is used to determine the maximum amount of stake that
+	/// can be unbonded for a period potentially lower than upper bound eras.
+	#[pallet::storage]
+	pub(crate) type EraLowestThirdTotalStake<T: Config> =
+		StorageMap<_, Twox64Concat, EraIndex, BalanceOf<T>>;
+
+	/// Parameters for the unbonding queue mechanism.
+	#[pallet::storage]
+	pub(crate) type UnbondingQueueParams<T: Config> = StorageValue<_, UnbondingQueue, ValueQuery>;
```

Comment:
```suggestion
	pub(crate) type UnbondingQueueParams<T: Config> = StorageValue<_, UnbondingQueue, OptionQuery>;
```

Maybe to not having a default and disable this feature when there is no parameter.
This requires a governance call to set the parameter to enable this feature, which I think should be the preferred way.

Reply:
Agreed 👍

Reply:
Amended to `OptionQuery` and added an unbonding params entry to `set_staking_configs`.